docs/usage/type-conversions.md

```diff
@@ -58,7 +58,7 @@ of the round trip conversion is the original object (in the sense that they live
 at the same memory address). There are a few exceptions:
 
 1. `NaN` is converted to `NaN` after a round trip but `NaN !== NaN`,
-2. `null` is converted to `undefined` after a round trip, and
+2. `null` is converted to `undefined` after a round trip
```

Comment:
it's generally considered to be good style to write numbered lists so that joining the entries together makes a grammatically correct sentence
```suggestion
2. `null` is converted to `undefined` after a round trip, and
```

Reply:
Thanks! I've learned one English writing style today 😃